src/orchestration/getOrchestrationClient.js

```diff
@@ -15,7 +15,7 @@ function getOrchestrationClient(mode) {
     case 'swarm':
       return SwarmDockerClient;
     case 'kubernetes':
-      return KubernetesClient;
+      return new KubernetesClient();
```

Comment:
This would be fine now, but later I all client classes should then be used as "non-static".
Maybe add an issue to remember it?

Reply:
I agree on principle (but yes different issue then) the problem is that the `listEngines` method does not use the `this` keyword (since it is not using anything from its class) and then the lint fails for having a class method without using `this`..

So we would have to disable that rule in that case. For me I dont care either way, disable the rule and make all instances of its classes, or have it as it is now with the lint rule in place